source/GameInterface/Services/ItemRosters/Patches/ItemRosterPatch.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Common.Logging;
+using Common.Messaging;
+using GameInterface.Services.ItemRosters.Messages.Commands.Internal;
+using HarmonyLib;
+using Serilog;
+using TaleWorlds.CampaignSystem.Roster;
+using TaleWorlds.Core;
+
+namespace GameInterface.Services.ItemRosters.Patches
+{
+    [HarmonyPatch(typeof(ItemRoster))]
+    internal class ItemRosterPatch
+    {
+        private static readonly ItemRosterPatch instance = new();
```

Comment:
Harmony patches do not require singletons, this is resolved using the harmony.PatchAll method

```suggestion
```

Reply:
I use it as a sender instance for the message broker.

Reply:
`__instance` is what we tend to use, It's more useful imo